include/zephyr/posix/sched.h

```diff
@@ -9,6 +9,7 @@
 #include <zephyr/kernel.h>
 
 #include "posix_types.h"
+#include "time.h"
```

Comment:
This is a standard ISO C header file, so it should have angle brackets and have a newline after private headers (sorry, this is not yet built into .clang-format)


```suggestion

#include <time.h>
```

Reply:
Noted, thank you !